src/server/routes/obvius.js

```diff
@@ -112,17 +112,23 @@ function obviusLog(req, res, next){
  * Verifies an Obvius request via username and password.
  */
 function verifyObviusUser(req, res, next){
-	// First we ensure that the password and email parameters are provided.
-	if (!req.param('password')) {
+	// First we ensure that the password and username parameters are provided.
+	const password = req.param('password');
+	// allowing for backwards compatibility if previous eGuage meters are using the 'email' parameter instead of
```

Comment:
First, indenting is off. Second, could you please add a TODO marker at start of line so it will stand out more and work with TODO systems.

Reply:
Yes, will do.